### PR TITLE
Searchtag Update

### PR DESCRIPTION
**Pull request by Kaedwuff**
Added a new "Search" (Orange with an S) tag to the security hub, and cleaned up the awful messy sprites that were in use for the criminal tags.

Sample here:

newtags

### CODE REVIEW

File: code/game/machinery/computer/security.dm [hunk 1]
```diff
@@ -104,6 +104,8 @@
 							switch(crimstat)
 								if("*Arrest*")
 									background = "'background-color:#DC143C;'"
+								if("Search")
+									background = "'background-color:##FFAE00;'"
```
Comment (Lohikar) [inline code/game/machinery/computer/security.dm]: `##FFAE00` should have one `#`.

File: code/game/machinery/computer/security.dm [hunk 2]
```diff
@@ -203,6 +205,8 @@
...
```
Comment (Lohikar) [inline code/game/machinery/computer/security.dm]: Same here.

**Comment by LifesDisciple** [comment]
is it just me or does it need a darker background? I can barely see the "S"

**Comment by Pacmandevil** [comment]
Yeah. doesn't look like it has the same depth as the other ones. and we already use a darker shade of yellow.

Perhaps a different colour?

**Comment by Kaedwuff** [comment]
How's this look? 

8996db6c69ee0c56172a22120afe95c1

**Comment by LifesDisciple** [comment]
slightly better. Mabe play with the colour of the letter? All the icons we have are white lettered. If you get the darker colour maybe you could make the light background work

**Comment by Kaedwuff** [comment]
If I use a different color than white for the letter then it won't match the aesthetic of the other ones. I mean it's pretty clearly visible right now so I don't see what the problem is.

**Comment by Kaedwuff** [comment]
That's not very feasible, Skull.  The R is like that because there isn't enough room in the P that is present. I'm not really sure you understand the size of the sprite we are working with here.  This is what it would look like if I adjusted the P's.

6d1fc2c586ea5687bf8ee27f0cd7c48d
bfe06856e3247b16f145faa3095fd567

Similarly, the S is a very complicated letter that requires an open space between two portions of it's structure.  The space I am working with is very limited.

53d3ec291a664d37f8e7ed8e4a600731

I could move it down a square..
be227155555814af4110af5ab405df62

Or I could thicken the S..

2402779cfd161484390da29dde9ba835

Or just make parts of it thicker than the rest...

bb7435b32ace1e6e3951153812bd0e40

But none of that really solves the problem without making it look even worse and more out of place.  I'm working with what I've got here.

**Comment by LordFowl** [comment]
@skull132 Response?

**Comment by skull132** [comment]
@Kaedwuff The second R, coupled with the second S should look better, if my perception is correct. At least they look more in line with the rest.

**Comment by LifesDisciple** [comment]
IMO they do not really need to be necessarily identical in hight as different letters are virtually never shown side to side

**Pull request merged**